### PR TITLE
Add documentation for zabbix item sensors

### PR DESCRIPTION
**Pull request by girlpunk**
**Description:**
Add documentation for zabbix item sensors.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#31006

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.zabbix.markdown [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
 title: "Zabbix Sensor"
-description: "Instructions on how to integrate Zabbix Triggers sensors within Home Assistant."
-date: 2016-12-13 22:57
+description: "Instructions on how to integrate Zabbix Triggers and Item sensors within Home Assistant."
+date: 2020-01-20 11:57
```
Comment (klaasnicolaas) [inline source/_components/sensor.zabbix.markdown]: Please do not change the date

Comment (frenck) [reply source/_components/sensor.zabbix.markdown]: Actually it should be removed 😉

Comment (girlpunk) [reply source/_components/sensor.zabbix.markdown]: As requested, I've added a patch to remove the date

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request closed without merging**